README.md

```diff
@@ -88,3 +88,32 @@ gauge('test').set_value(42)
 ```
 
 See `examples/pyformance_usecase.py` for a complete code example using Pyformance.
+
+## Known Issues
+
+1. Sending only 1 datapoint and not seeing it in the chart.
+
+Root Cause: The reason you are not seeing the metrics in the chart is because the script that is calling the python client module is exiting right after calling the send method. The python client library is mainly targeted towards sending a continuous stream of metrics and was implemented to be asynchronous.
+
+Workaround:  Adding a sleep [eg: time.sleep(5)] for say 5 secs before exciting from your script or run your script from a python interpreter you should start seeing your metric in the chart. Or if you send a stream or metrics, you will see the metrics in the chart.?add
+
+
+2. SSLError when sending events by calling send_event() method
```

Comment:
That's a pretty bad practice that you are advocating as a work around. There should be a way to gracefully shutdown the `SignalFx` instance, such that it no longer accepts new data, but will drain the current queue, with a way to wait for the queue to be fully drained before returning.